ui-v2/app/models/gateway.js

```diff
@@ -0,0 +1,12 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+
+export const PRIMARY_KEY = 'uid';
+export const SLUG_KEY = 'Name';
+export default Model.extend({
+  [PRIMARY_KEY]: attr('string'),
+  [SLUG_KEY]: attr('string'),
+  Datacenter: attr('string'),
+  Namespace: attr('string'),
+  Services: attr(),
```

Comment:
I think the reason why I was quibbling over this Model is that it has a property that is a collection of models we already have. This is fine for now but I'm thinking there could be another way to do this.